content/applications/general/iot/windows_iot.rst

```diff
@@ -0,0 +1,158 @@
+#. Set up GPL Ghostscript: Click :guilabel:`Next`, agree to the terms and conditions, click
+   :guilabel:`Install`, then :guilabel:`Finish`.
+#. Click :guilabel:`Next`, :guilabel:`Next`, and :guilabel:`Finish` to complete the setup.
+
+The :ref:`IoT system's homepage <iot/windows-iot/homepage>` automatically opens in a web browser
+with the URL `http://localhost:8069`.
+
+.. tip::
+   If the web browser does not show anything, :ref:`restart <iot/windows_iot/restart>` the Windows
+   virtual IoT service.
+
+.. _iot/windows-iot/firewall:
+
+Windows Firewall configuration
+==============================
+
+Firewalls help keep devices secure but can sometimes block legitimate connections. If the Windows
+virtual IoT isn't accessible on the :abbr:`LAN (Local Area Network)`, for example from another
+device, it could be due to a firewall blocking the connection. To prevent this issue, configure
+exceptions for network discovery in the the :abbr:`OS (Operating System)` or firewall settings.
+
+.. note::
+   If third-party firewall software is installed on the Windows computer, refer to the software's
+   documentation to configure firewall exceptions.
+
+To create a rule on Windows Defender and allow communication through port `8069`, follow these
+steps:
+
+#. Search the Windows start menu for `firewall` and select the :guilabel:`Windows Defender Firewall
+   with Advanced Security` app.
+#. In the left part of the window, select :guilabel:`Inbound Rules`.
+#. In the right part of the window, under :guilabel:`Actions`, click :guilabel:`New Rule`.
+#. In the :guilabel:`New Inbound Rule Wizard` that opens, select the :guilabel:`Port` type of rule
+   and click :guilabel:`Next`.
+#. On the :guilabel:`Protocols and Ports` page, make sure :guilabel:`TCP` and :guilabel:`Specified
+   local ports` are selected, enter the following in the field: `8069, 80, 443`, and click
+   :guilabel:`Next`.
+
+   .. note::
+      Other ports may be necessary depending on your IoT devices. For example, for the
+      :doc:`/applications/sales/point_of_sale/payment_methods/terminals/worldline` payment terminal,
+      add the `9050` port.
+
+#. On the :guilabel:`Action` page, select :guilabel:`Allow the connection` and click
+   :guilabel:`Next`.
+#. On the :guilabel:`Profile` page, disable any connection type(s) that don't apply to your
+   Windows computer and click :guilabel:`Next`.
+#. On the :guilabel:`Name` page, provide a :guilabel:`Name` (e.g., `Odoo`) and, optionally, a brief
+   :guilabel:`Description`, then click :guilabel:`Finish`.
+
+.. seealso::
+   `Windows Firewall rules documentation
+   <https://learn.microsoft.com/en-us/windows/security/operating-system-security/network-security/windows-firewall/rules>`_
+
+.. _iot/windows-iot/homepage:
+
+Windows virtual IoT homepage
+============================
+
+To access the Windows virtual IoT's homepage, open a web browser **on the same network as the IoT
+system** and navigate to the URL `http://localhost:8069`.
```

Comment:
misleading `localhost:8069` will only work on the windows IoT computer. From another device in the LAN, it should be `<iot ip address>:8069`